src/python/pants/core_tasks/generate_pants_ini.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import configparser
+import logging
+import os
+import sys
+
+from pants.base.exceptions import TaskError
+from pants.task.task import Task
+from pants.version import VERSION
+
+
+logger = logging.getLogger(__name__)
```

Comment:
This may work, but for a v1 Task, `self.context.log` is canonical and widespread.